docs/paper/admin/reference/configuration/server-properties.mdx

```diff
@@ -0,0 +1,224 @@
+initial-enabled-packs:
+  default: 'vanilla'
+  description: 'Comma-separated list of datapacks to be enabled during world creation. Feature packs need
+    to be explicitly enabled.'
+level-name:
+  default: 'world'
+  description: 'The name of the world. This will be the name of the folder in which the world is saved.'
+level-seed:
+  default: ''
+  description: 'The seed used to generate the world. Leave blank to default to random.'
+level-type:
+  default: 'minecraft\\:normal'
+  description: 'Defines the type of the world generator. (Allowed values: "normal", "flat", "large_biomes",
+    "amplified", "single_biome_surface", "buffet", "default_1_1", "customized")'
+max-chained-neighbor-updates:
+  default: '1000000'
+  description: 'Limits the number of consecutive neighbour updates before skipping subsequent updates. Negative
+    values will disable the limit.'
+max-players:
+  default: '20'
+  description: 'The maximum number of players that can play on the server at the same time.'
+max-tick-time:
+  default: '60000'
+  description: 'The maximum number of milliseconds a single tick may take before the server watchdog stops the
+    server with the message. If a single server tick took 60.00 seconds (should be max 0.05) it will be considered to be
+    crashed and the server will forcibly shutdown via calling System.exit(1). Setting this to -1 will disable watchdog
+    entirely.'
+max-world-size:
+  default: '29999984'
+  description: 'The maximum allowed size of the world radius, in blocks. This only affects the chunks that are
+    generated when the world is initially created, and not the world border. (Limited to 29999984)'
+motd:
+  default: 'A Minecraft Server'
+  description: 'The message of the day, displayed in the server list.'
+network-compression-threshold:
+  default: '256'
+  description: 'The number of bytes of a packet before it is compressed. Setting to a negative disables compression.'
+online-mode:
+  default: 'true'
+  description: 'If set to true, the server checks all connecting players against Minecraft''s account database.
+    This requires all connected players to have a valid Minecraft account and makes it impossible for cracked
+    players to connect.'
+op-permission-level:
+  default: '4'
+  description: 'Sets the default permission level for ops when using /op. (Allowed values: 0, 1, 2, 3, 4)'
+player-idle-timeout:
+  default: '0'
+  description: 'If non-zero, players are kicked from the server if they are idle for more than that many minutes.
+    (Default: 0). The following packets stop this timer: "Click Window", "Enchant Item", "Update Sign",
+    "Player Digging", "Player Block Placement", "Held Item Change", "Animation (swing arm)", "Entity Action",
+    "Client Status", "Chat Message", "Use Entity"'
+prevent-proxy-connections:
+  default: 'false'
+  description: 'If the ISP/AS sent from the server is different from the one from Mojang Studios'' authentication
+    server, the player is kicked.'
```

Comment:
kicked implies they were able to join successfully, rather than prevented like the name suggests